src/NuGet.Core/NuGet.Packaging/Rules/InstallScriptInPackageReferenceProjectRule.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace NuGet.Packaging.Rules
 {
-    class InstallScriptInPackageReferenceProjectRule : IPackageRule
+    internal class InstallScriptInPackageReferenceProjectRule : IPackageRule
```

Comment:
see this [comment](https://github.com/NuGet/NuGet.Client/pull/3226#discussion_r377183570)

Reply:
[Rule #5](https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md)